lib/typeInference.js

```diff
@@ -9,36 +9,46 @@ const jsFunctions = require('./jsFunctions')
 const isEmptyObj = require('./utilityFunctions').isEmptyObj
 const globalTypesObj = {}
 
-const isFunction = (id, typeObj) => typeObj[id] !== undefined && typeObj[id] !== null && typeObj[id].type === 'function'
+const isFunction = (id, typeObj) =>
+  typeObj[id] !== undefined &&
+  typeObj[id] !== null &&
+  typeObj[id].type === 'function'
 
-const getParamReturnTypes = (id, typeObj) => [typeObj[id.name].paramTypes, typeObj[id.name].returnType]
+const getParamReturnTypes = (id, typeObj) => [
+  typeObj[id.name].paramTypes,
+  typeObj[id.name].returnType
+]
 
-const isEmpty = arr => arr.toString() === ''
+const isEmpty = (arr) => arr.toString() === ''
 
 const returnIdentifierType = (id, typeObj) => {
   if (isFunction(id.name, typeObj)) return getParamReturnTypes(id, typeObj)
   return typeObj[id.name]
 }
 
 const identifierType = (id, localTypeObj) => {
-  if (localTypeObj[id.name] !== undefined) return returnIdentifierType(id, localTypeObj)
-  if (globalTypesObj[id.name] !== undefined) return returnIdentifierType(id, globalTypesObj)
+  if (localTypeObj[id.name] !== undefined) { return returnIdentifierType(id, localTypeObj) }
```

Comment:
Please change the eslint rule `curly` to `multi` instead of `all`

Reply:
You can modify the `eslint` config in `package.json` to add this:
```json
{
  "eslint" : {
    "curly": "multi"
  }
}
```
There are options provided for some rules, you can check the docs for this specific rule [here](https://eslint.org/docs/rules/curly).

Similarly, there might be rule options that allow us not to change all `let` bindings to `const`.

Reply:
@angadgill92 I am sorry if I am missing something. I added the rules but saw no effect. 
`standard` says we can not customize rules. Please [See here](https://standardjs.com/#i-disagree-with-rule-x-can-you-change-it). For customizing rules, it says to use [`standardX`](https://github.com/standard/standardx).
Please correct me if I am missing anything.

Reply:
My bad, the config actually goes in `.eslintrc` let's leave it for now though, we can take it up later if required 👍🏼